src/lib.rs

```diff
@@ -141,6 +141,11 @@ impl ContextBuilder {
 
 /// Context is a wrapper around a QuickJS Javascript context.
 /// It is the primary way to interact with the runtime.
+///
+/// For each `Context` instance a new instance of QuickJS
+/// runtime is created. It means that it is safe to use
+/// different contexts in different threads, but each
+/// `Context` instance shuld be used only from a single thread.
```

Comment:
slight correction: each `Context` **must** only be used from a single thread, since it is `!Send`.

Reply:
I've removed part about the threads at all, as it follows from the fact that `Send` and `Sync` traits are not implemented.

Reply:
Oh I didn't mean to remove the whole note.
I think it's informative for the user!

Just replacing should with must is fine. ;)